tests/end-to-end/grafana/admin-promotion-ops.cy.js

```diff
@@ -32,7 +32,7 @@ describe('ops grafana user promotion', function () {
 
   afterEach(function () {
     cy.clearAllCookies()
-    cy.wrap(null).then(Cypress.session.clearAllSavedSessions)
+    cy.wrap(-1).then(Cypress.session.clearAllSavedSessions)
```

Comment:
Why is wrap needed here? Why not just call it like this:
```suggestion
    cy.then(Cypress.session.clearAllSavedSessions)
```

Reply:
But how would the `wrap` function impact any of that? I just looked up the [documentation for the wrap function](https://docs.cypress.io/api/commands/wrap) and couldn't understand how it would make any difference here, that's why I wondered if was needed or if it had any reason to be there previously.